apps/reference/docs/guides/auth/auth-ui/auth-ui-customization.mdx

```diff
@@ -0,0 +1,254 @@
+---
+id: auth-ui-customization
+title: Customization
+description: Securing your user data with Row Level Security.
+---
+
+# Customizing Auth UI
+
+Auth UI is a React component that provides a UI that 'works out of the box'.
+
+There are several ways to customize Auth UI:
+
+- Using one of the [predefined Themes](#themes) that comes with Auth UI.
+- Extending a theme by [overriding the variable tokens](#overwriting-themes) in a theme.
+- [Creating your own Theme](#creating-your-own-theme)
+- [Using your own CSS classes](#using-your-own-css-classes)
+- [Using inline styles](#using-your-own-inline-styles)
+- [Using your own labels](#using-your-own-custom-labels)
+
+Before proceeding, you may want to already be familiar with the [installation of Auth UI](/docs/guides/auth/auth-ui/auth-ui-install).
+
+---
+
+## Themes
+
+### Predefined themes
+
+Auth UI comes with several themes that you can use to customize the look and feel of Auth UI.
+
+Each predefined theme comes with at least 2 variations, a `default` variation, and a `dark` variation. You are able to switch between these themes by using the `theme` prop.
+
+Import the theme you want to use and pass it to the `appearence.theme` prop.
+
+```js title="/src/index.js"
+import { createClient } from '@supabase/supabase-js'
+import { Auth, ThemeSupa } from '@supabase/react-ui-react'
+
+const supabase = createClient(
+  '<INSERT PROJECT URL>',
+  '<INSERT PROJECT ANON API KEY>'
+)
+
+const App = () => (
+  <Auth supabaseClient={supabase} appearance={{ theme: ThemeSupa }} />
+)
+```
+
+:::info
+
+Currently there is only 1 predefined theme available, but we plan to add more.
+
+:::
+
+### Switching theme variations
+
+Auth UI comes with 2 theme variations: `default` and `dark`. You can switch between these themes by using the `theme` prop.
+
+```js title="/src/index.js"
+import { createClient } from '@supabase/supabase-js'
+import { Auth, ThemeSupa } from '@supabase/react-ui-react'
+
+const supabase = createClient(
+  '<INSERT PROJECT URL>',
+  '<INSERT PROJECT ANON API KEY>'
+)
+
+const App = () => (
+  <Auth
+    supabaseClient={supabase}
+    appearance={{ theme: ThemeSupa }}
+    theme="dark"
+  />
+)
+```
+
+The default of the theme prop is always `"default"`, but you can also pass `"dark"` to the theme prop to switch to the `dark` theme. If you theme has other variations, just use the name of the variation in this prop to switch it to that theme variation.
```

Comment:
```suggestion
If you don't pass a vale to `theme` it will use the `"default"` theme. You can pass `"dark"` to the theme prop to switch to the `dark` theme. If your theme has other variations, use the name of the variation in this prop.
```